### PR TITLE
Added tests for backend part

### PR DESCRIPTION
**Pull request by dmitryk-dk**
Added tests to the backend part of the plugin

### CODE REVIEW

**Review by Loori-R — APPROVED** [review]
LGTM!

File: pkg/plugin/response_test.go [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+	"github.com/grafana/grafana-plugin-sdk-go/data"
+)
+
+func Test_parseStreamResponse(t *testing.T) {
+	f := func(remoteResp string) io.Reader {
+		return io.NopCloser(bytes.NewBuffer([]byte(remoteResp)))
+	}
+	tests := []struct {
+		name     string
+		reader   func(remoteResp string) io.Reader
```
Comment (hagen1778) [inline pkg/plugin/response_test.go:21]: `reader` seems identical in every test case. Do you need it?

Comment (dmitryk-dk) [reply pkg/plugin/response_test.go]: Removed it as a param of the test struct

**Review by hagen1778 — APPROVED** [review]
lgtm

**Pull request merged**